.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: CI
+
+on:
```

Comment:
Add excludes for files:
- .editorconfig
- .gitignore
- '*.md'
- 
so the build won't be triggered when noncode files are changed